File: ASMR.Web/ClientApp/src/data/release/common/AndroidReleaseInformation.ts
import AppReleaseInformation from "@asmr/data/release/common/AppReleaseInformation"
import StoreReleaseInformation from "@asmr/data/release/common/StoreReleaseInformation"

interface AndroidReleaseInformation extends AppReleaseInformation {
	PlayStore: StoreReleaseInformation
}

export default AndroidReleaseInformation

File: ASMR.Web/ReleaseInformation/Common/AndroidReleaseInformation.cs
//
// asmr: Coffee Beans Management Solution
// © 2021 Pandora Karya Digital. All right reserved.
//
// Written by Danang Galuh Tegar Prasetyo [connect@danang.id]
//
// AndroidReleaseInformation.cs
//

using System.Text.Json.Serialization;

namespace ASMR.Web.ReleaseInformation.Common;

public class AndroidReleaseInformation : AppReleaseInformation
{
	[JsonPropertyName("PlayStore")] public StoreReleaseInformation PlayStore { get; set; }
}